node/request.js

```diff
@@ -110,6 +111,31 @@ TChannelRequest.prototype.hookupCallback = function hookupCallback(callback) {
     function onError(err) {
         if (called) return;
         called = true;
+
+        if (err.isErrorFrame) {
+            self.channel.emitStat(new Stat.Counter(
+                'outbound.calls.systems-errors', 1, {
+                    'target-service': self.serviceName,
+                    'service': self.headers.cn,
+                    // TODO should always be buffer
+                    'target-endpoint': String(self.arg1),
+                    'error-type': err.codeName
```

Comment:
:)  I have no strong preference on the names (I kindof like yours better tbh) - just want to make sure that we record whatever changes we make.